src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_RADIUS = 10.0;
+    private static final int NUMBER_OF_FIGURE = 5;
+    private static final double MAXIMUM_VALUE = 50.0;
+    private final Circle defaultFigure = new Circle(Color.WHITE.name()
+            .toLowerCase(), DEFAULT_RADIUS);

```

Comment:
weird formatting

why not declaring it in a single line?
```suggestion
    private final Circle defaultFigure = new Circle(Color.WHITE.name().toLowerCase(), DEFAULT_RADIUS);

```